scripts/config-build.py

```diff
@@ -107,7 +114,11 @@ def setup_ats(scriptsdir, buildpath):
 if unknown_args:
     print("[config-build]: Passing the following unknown arguments directly to cmake... %s" % unknown_args)
 
+if args.buildpath != "" and args.buildrootdir != "" :
+    sys.exit("Buildpath and buildrootdir are not compatible, you may spectify full buildpath with --buildpath (-bp)")
 
+if args.installpath != "" and args.installrootdir != "" :
+    sys.exit("Installpath and installrootdir are not compatible, you may spectify full installpath with --installpath (-ip)")
```

Comment:
Couldn't https://docs.python.org/3/library/argparse.html#mutual-exclusion be used instead? It looks built-in the std package.